client/app/bundles/course/assessment/question/programming/reducers/utils.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Formats the display for file sizes
+ */
+export default function formatBytes(bytes, decimals) {
```

Comment:
I'll generally prefer not to have defaults for `utils.js`, because the requiring then becomes implicit. If possible we can silence the linter here for this purpose.